app/controllers/concerns/inherited_proofing_concern.rb

```diff
@@ -5,6 +5,16 @@
 module InheritedProofingConcern
   extend ActiveSupport::Concern
 
+  include Idv::InheritedProofing::ServiceProviders
+
+  def inherited_proofing?
+    inherited_proofing_service_provider.present?
+  end
+
+  def inherited_proofing_service_provider
+    return VA if va_inherited_proofing?
```

Comment:
Just to keep simpifying, and removing indirection, I feel moderately strongly we should get rid of this `VA` constant (and the module that defines it). A symbol is basically a constant so just inlining `:va` is fine here:

```suggestion
    return :va if va_inherited_proofing?
```

Reply:
Done.